Common/Models/Currency/CurrencyModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Common.Models.User;
+using Postgrest.Attributes;
+
+namespace Common.Models.Currency
+{
+    [Table("Currencies")]
+    public class CurrencyModel : BaseDbModel
+    {
+        [PrimaryKey("Id")]
+        public int Id { get; set; }
+
+        [Column("Value")]
+        public string Value { get; set; }
+
+        public virtual ICollection<UserModel> Users { get; set; } = new HashSet<UserModel>();
```

Comment:
Why is this and several other attributes in other models Virtual? That means they can be overwritten and I really do not like the Idea of that. Sounds rather risky and sure as hell not secure way to handle this.